src/components/ComputeModal.js

```diff
@@ -975,6 +1013,37 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
           ])
         ])
       ]),
+      div({ style: { gridColumnEnd: 'span 6', marginTop: '1.5rem' } }, [
+        h(LabeledCheckbox, {
+          checked: computeConfig.autopause,
+          disabled: !autoPauseCheckboxEnabled,
+          onChange: v => updateComputeConfig('autopause', v)
+        }, [
+          span({ style: { marginLeft: '0.5rem', ...computeStyles.label, verticalAlign: 'top' } }, [
+              enableAutoPauseSpan
+          ]),
+          h(Link, {
+            style: { marginLeft: '1rem', verticalAlign: 'top' },
+            href: 'https://support.terra.bio/hc/en-us/articles/360029761352-Preventing-runaway-costs-with-Cloud-Environment-autopause-#h_27c11f46-a6a7-4860-b5e7-fac17df2b2b5', ...Utils.newTabLinkProps
+          }, [
+            'Learn more about autopause.',
+            icon('pop-out', { size: 12, style: { marginLeft: '0.25rem' } })
+          ])
+        ]),
+        div({ style: { ...gridStyle, gridTemplateColumns: '0.75fr 4.5rem 1fr 5rem 1fr 5rem', marginTop: '0.75rem' } }, [
+          h(NumberInput, {
+            min: 1,
+            max: 999,
+            isClearable: false,
+            onlyInteger: true,
+            value: computeConfig.autopauseThreshold,
+            disabled: !computeConfig.autopause,
+            tooltip: !computeConfig.autopause ? 'Autopause must be enabled to configure pause time.' : undefined,
+            onChange: updateComputeConfig('autopauseThreshold')
+          }),
+          span({style: {width: '300px'}}, ['minutes of inactivity'])
+        ])
```

Comment:
```suggestion
        div({ style: { ...gridStyle, gridGap: '0.8rem', gridTemplateColumns: '4.5rem 9.5rem', marginTop: '0.75rem' } }, [
          h(NumberInput, {
            min: 1,
            max: 999,
            isClearable: false,
            onlyInteger: true,
            value: computeConfig.autopauseThreshold,
            disabled: !computeConfig.autopause,
            tooltip: !computeConfig.autopause ? 'Autopause must be enabled to configure pause time.' : undefined,
            onChange: updateComputeConfig('autopauseThreshold')
          }),
          span(['minutes of inactivity'])
        ])
```


would achieve the following:
- remove unnecessary `gridTemplateColumns` since only the first two are relevant in this case (because we only have the number box and the text);
- use `rem` for both so it's easier to compare, etc.;
- override `gridGap` inherited from parent (through `…gridStyle`) to shorten it, which allows addressing the UX request to move the text closer to the number box. I put `0.8rem` but I'd check with UX to be sure;
- get rid of the custom `style`ing in `span()` since it's taken care of by the previous changes and it's good practice to minimize lower-level `style`ing.

With that change, the gap looks like the following:

![image](https://user-images.githubusercontent.com/5438223/155230504-6c363056-ac54-4d8e-bde7-f726960fae41.png)